core/src/main/java/org/elasticsearch/tasks/Task.java

```diff
@@ -89,7 +89,7 @@ public final TaskInfo taskInfo(String localNodeId, boolean detailed) {
     /**
      * Build a proper {@link TaskInfo} for this task.
      */
-    protected final TaskInfo taskInfo(String localNodeId, String description, Status status) {
+    public final TaskInfo taskInfo(String localNodeId, String description, Status status) {
```

Comment:
Yeah, I didn't want to do this but couldn't think of a good way since the caller isn't part of that type hierarchy anymore. I'll look for a better alternative